tfhe/src/core_crypto/entities/lwe_ciphertext.rs

```diff
@@ -688,6 +689,23 @@ pub type LweCiphertextView<'data, Scalar> = LweCiphertext<&'data [Scalar]>;
 /// An [`LweCiphertext`] mutably borrowing memory for its own storage.
 pub type LweCiphertextMutView<'data, Scalar> = LweCiphertext<&'data mut [Scalar]>;
 
+pub struct LweCtParameters<T: UnsignedInteger> {
+    pub lwe_dim: LweDimension,
+    pub ct_modulus: CiphertextModulus<T>,
+}
+
+impl<C: Container> ParameterSetConformant for LweCiphertext<C>
+where
+    C::Element: UnsignedInteger,
+{
+    type ParameterSet = LweCtParameters<C::Element>;
+
+    fn is_conformant(&self, lwe_ct_parameters: &LweCtParameters<C::Element>) -> bool {
+        lwe_ct_parameters.lwe_dim.to_lwe_size() == self.lwe_size()
+            && lwe_ct_parameters.ct_modulus == self.ciphertext_modulus()
+    }
```

Comment:
forgot to tell you we have a `check_content_respects_mod` in tfhe/src/core_crypto/algorithms/test/mod.rs that we can move to tfhe/src/core_crypto/algorithms/misc.rs

and check that the content in the container is valid like you can see in https://github.com/zama-ai/tfhe-rs/blob/2e8189514c9e80aa4542c8b795c96f656237edd3/tfhe/src/core_crypto/algorithms/test/lwe_encryption.rs#L189

this only requires the input entity to be as_ref which all our entities are